Makefile

```
@@ -84,12 +88,17 @@ install: bdist lint ## Build distribution and install
 	jupyter serverextension list
 	jupyter labextension list
 
-test: lint ## Run unit tests
+test-server: lint-server ## Run unit tests
 	pytest -v elyra
 
-test-ui: lint ## Run frontend tests
+test-ui: lint-ui ## Run frontend tests
 	npm test
 
+test-ui-debug: lint-ui ## Run frontend tests
```

```suggestion
test-ui-debug: lint-ui
```
probably should remove the comment here (my bad)